ldk-net/ldk_net.c

```diff
@@ -214,7 +214,10 @@ static void *sock_thread_fn(void* arg) {
 								if (newfd >= 0) {
 									// Received a new connection, register it!
 									LDKSocketDescriptor new_descriptor = get_descriptor(handler, newfd);
-									LDKCResult_NonePeerHandleErrorZ con_res = PeerManager_new_inbound_connection(&handler->ldk_peer_manager, new_descriptor);
+									LDKCOption_NetAddressZ remote_network_address = {
+										.tag = LDKCOption_NetAddressZ_None,
```

Comment:
nit: calling `COption_NetAddressZ_none()` is a lot cleaner, and maybe can even be inlined in the method call.

Reply:
A bit of a novice regarding socket programming. Is there a way to determine if `sockaddr_in` or `sockaddr_in6` should be passed in for a given file descriptor?